### DISCUSSION TITLE
Hi , Trying to add more languages to the language bar , it shows only 5 top languages

### DISCUSSION THREAD

**Discussion by iDereje** [anuraghazra/github-readme-stats#2733] [Q&A] [answered]
Hi ,

I've updated my "Settings ", but nothing changes , Anyone countered the same issue .

 <a href=""> <img align="center" src="https://github-readme-stats-sigma-five.vercel.app/api/top-langs/?username=iDereje&theme=react&line_height=40&hide=css&count_private&langs_count=10"/> </a>

**Comment by qwerty541** [comment, marked as answer]
Looks like the source of your problem is outdated vercel deployment. I have tested on my own vercel instance which is up to date with master branch and there is no problem there:

`https://github-readme-stats-steel-omega.vercel.app/api/top-langs/?username=iDereje&langs_count=10`

<img src="https://github-readme-stats-steel-omega.vercel.app/api/top-langs/?username=iDereje&langs_count=10" />

**Comment by iDereje** [comment]
thanks alexander i've fixed the issue

**Reply by qwerty541** [reply]
Nice! Can you mark this discussion as answered please?
